src/app/components/ListForm/ListForm.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import ListForm from '.';
```

Comment:
this import wont work

Reply:
The import works, but thanks to this comment I noticed that I am missing a Provider 🥲 
Storybook with Redux can#t work without one, so thank you very much 🎉